frontend/src/components/SearchImage/SearchImage.jsx

```diff
@@ -1,28 +1,49 @@
-import React, { useRef } from 'react'
-import './SearchImage.scss';
+import React, { useRef, useState, useEffect } from "react";
+import "./SearchImage.scss";
 
 const SearchImage = ({ onChange, setInputValue, value }) => {
-    const searchRef = useRef();
+  const searchRef = useRef();
+  const [isSearchBarFull, setSearchBarFull] = useState(true);
 
-    function handleSearchSubmit(e) {
-        e.preventDefault();
-        const searchTerm = searchRef.current.value;
-        onChange(searchTerm);
+  function handleSearchSubmit(e) {
+    e.preventDefault();
+    const searchTerm = searchRef.current.value;
+    onChange(searchTerm);
+  }
+
+  const isSticky = () => {
+    const scrollTop = window.scrollY;
+    if (scrollTop > 20) {
+      setSearchBarFull(false);
+    } else {
```

Comment:
this all if statement can be shortened since its only toggles true/false. 
Ex: 
`setSearchBarFull(scrollTop <= 20);`

Reply:
resolve

Reply:
did some improvements here